### PR TITLE
[docs] Update the website 5/25/2020

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
Merge to update the website.

Preview at: https://vasilystrelyaev.github.io/testcafe/.

* Added blog post notifications (to bring them back again and again for testing purposes, delete `team-blog-last-visited` from the local storage).
* Added new @SergeyShurygin's  post: [How to Debug Tests in TestCafe: Quick Guide](https://vasilystrelyaev.github.io/testcafe/media/team-blog/how-to-debug-tests-in-testcafe-quick-guide.html)
* Fixed and updated [Community Blog](https://vasilystrelyaev.github.io/testcafe/media/community/) and [Courses](https://vasilystrelyaev.github.io/testcafe/media/courses/)

### CODE REVIEW

**Comment by alexphilin** [comment]
> It is correct? /сс @alexphilin
> 
> Снимок экрана 2020-05-26 в 09 06 21

Looks bad.

**Comment by helen-dikareva** [comment]
And in mobile menu:
image

File: blog/introducing-testcafe-open-source-testing-framework.html [hunk 1]
```diff
@@ -251,6 +251,27 @@
...
+<script>
+    var TEAM_BLOG_LAST_VISITED_FLAG = 'team-blog-last-visited';
+    var TEAM_BLOG_LINK_ID           = 'blog-link';
+    var TEAM_BLOG_LINK_NEW_CLASS    = 'new-post';
+    var NOTIFICATIONS_ACTIVE_SINCE  = new Date(2020, 4, 21)
+
+    var teamBlogLastVisited = window.localStorage.getItem(TEAM_BLOG_LAST_VISITED_FLAG) || NOTIFICATIONS_ACTIVE_SINCE;
+
+    window.addEventListener('DOMContentLoaded', function (e) {
+        var lastPostDate = new Date(1590354000 * 1000);
```
Comment (helen-dikareva) [inline blog/introducing-testcafe-open-source-testing-framework.html:265]: Better to use more readable format for this date and even extract it to a const

**Comment by VasilyStrelyaev** [comment]
image

in all browsers

File: blog/introducing-testcafe-open-source-testing-framework.html (hunk 1 above)
Comment (VasilyStrelyaev) [reply blog/introducing-testcafe-open-source-testing-framework.html:265]: It looks like that in code:

https://github.com/DevExpress/testcafe-gh-page-assets/blob/master/src/_includes/header.html#L65

You're looking at the complied site

Comment (helen-dikareva) [reply blog/introducing-testcafe-open-source-testing-framework.html:265]: ok, sure

**Comment by VasilyStrelyaev** [comment]
This should fix it, however, one moment pls, Sergey wants to add a new paragraph to the blog post now.

**Comment by VasilyStrelyaev** [comment]
Checked the new sentence with Dirk, ready to review.

**Pull request merged**